tests/vm_test.go

```diff
@@ -1921,6 +1922,91 @@ status:
 			}, 2*time.Minute, 1*time.Second).Should(Succeed())
 		})
 	})
+
+	Context("when node becomes unhealthy", Serial, func() {
+		var nodeName string
+
+		AfterEach(func() {
+			node, err := kubevirt.Client().CoreV1().Nodes().Get(context.Background(), nodeName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			node.Spec.Unschedulable = false
+			node, err = kubevirt.Client().CoreV1().Nodes().Update(context.Background(), node, metav1.UpdateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(node.Spec.Unschedulable).To(BeFalse())
+
+			libnode.Untaint(nodeName, "node.kubernetes.io/unschedulable")
+		})
+		It("[Serial] the VMs running in that node should be respawned", func() {
+			var pod *k8sv1.Pod
+			var wg sync.WaitGroup
+
+			By("Starting VM")
+
+			vmi := libvmi.NewAlpine()
+			vmi.Namespace = testsuite.GetTestNamespace(vmi)
```

Comment:
Not necessary, you can inline `libvmi.NewAlpine` into `createRunningVM` and drop this line.

Reply:
Done